packages/studio-be/src/studio/studio-router.ts

```diff
@@ -210,6 +215,7 @@ export class StudioRouter extends CustomRouter {
               window.BOT_LOCKED = ${!!bot.locked};
               window.WORKSPACE_ID = "${workspaceId}";
               window.IS_BOT_MOUNTED = ${this.botService.isBotMounted(botId)};
+              window.IS_CLOUD_BOT = ${!bot.standalone}
```

Comment:
standalone !== cloud bot, we'll have to double check this moreover it doesn't really have anything to do with code editor ?

Reply:
i'll change it, it will be a cloud flag instead